src/app/features/lp-form/thunks.ts

```diff
@@ -176,6 +175,18 @@ export const getInfoPostLpThunk = createAsyncThunk<
 >('lpForm/getInfoPostLp', async (_, thunkAPI) => {
   try {
     const lpFormState = thunkAPI.getState().lpForm;
+
+    const fixedLower = lpFormState.userInput.fixedLower;
+    const fixedUpper = lpFormState.userInput.fixedUpper;
+
+    if (!fixedLower || !fixedUpper) {
```

Comment:
when fixedLower === 0 and fixedUpper === 0 this will execute. Is this intended or you meant to check for null/undefined?